packages/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utilities';
+import { ColorTokens, RecursivePartial, FontTokens } from '@fluentui/react-theme-provider/lib/compat/index';
```

Comment:
```suggestion
```

We should not have any dependencies on `/compat` folders. As we don't support `tokens` yet, these types can be removed safely.

Reply:
Updated